CONTRIBUTING.md

```diff
@@ -69,5 +76,41 @@ To test the examples, run the Nox `examples` session:
     $ nox -s examples
 ```
 
+
+This will test all scripts within the `examples` directory.
+To only test one script:
+
+```console
+    $ nox -s examples -- <script_name>.py
+```
+
 For more information on developing examples, see the examples
 [README.md](examples/README.md)
+
+
+## Documentation
+
+Documentation is built from Markdown files in the `docs` directory using
+[MkDocs](https://www.mkdocs.org/) according to `mkdocs.yml`. The API reference
+is auto-populated from code docstrings. These docstrings must be in the
+[google format](https://mkdocstrings.github.io/handlers/python/#google-style)
+(note: we use `Parameters` in our docstrings).
+
+Nox is used to manage the process of building the documentation as well as and
+hosting it locally to assist documentation development.
```

Comment:
"as well as [serving?] and hosting" maybe?

Reply:
I thought hosting included serving. It doesn't? This is new to me! What is the difference between serving and hosting?

Reply:
oh I see now, there is an extra "and"! I'm actually going to use "as well as serving it locally..." because the actual command is `mkdoc serve`